Header header

uint64 handle
uint64 serial_number
string model_name
string firmware_version

float32 last_reported_temperature
float32 last_reported_humidity
float32 last_reported_age

uint8 gps_ts_year
uint8 gps_ts_month
uint8 gps_ts_day
uint8 gps_ts_hour
uint8 gps_ts_min
uint8 gps_ts_sec
